modules/serverless-logic-workflow-passing-arguments-internal-builder.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-workflow-passing-arguments-internal-builder_{context}"]
+= Passing arguments to the internal builder
+
+You can pass build arguments to the `SonataFlowBuild` instance or set default build arguments in the `SonataFlowPlatform` resource, to allow customization of the build process.
+
+.Prerequisites
+
+* You have installed the {ServerlessLogicOperatorName} on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
+* You have created your {ServerlessLogicProductName} project.
+* You have installed the OpenShift CLI `(oc)`.
+
+.Procedure
+
+. Check for the existing `SonataFlowBuild` instance, by running the following command:
++
+[source,terminal]
+----
+$ oc get sonataflowbuild <name> -n <namespace> <1>
+----
++ 
+<1> Replace `<name>` with the name of your `SonataFlowBuild` instance and `<namespace>` with your namespace.
+
+. Add build arguments to the `SonataFlowBuild` instance, by running the following command:
++
+[source,terminal]
+----
+$ oc edit sonataflowbuild <name> -n <namespace>
+----
+
+. Add the desired build arguments under the `.spec.buildArgs` field of the `SonataFlowBuild` instance:
++
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowBuild
+metadata:
+  name: <name>
+spec:
+  buildArgs:
+    - name: ARG1
+      value: value1
+    - name: ARG2
+      value: value2
+----
```

Comment:
@kaldesai Feedback not implemented. Please check this feedback and implement: https://github.com/openshift/openshift-docs/pull/76490#discussion_r1666837459

Reply:
Apologies! I added now